lib/main.js

```diff
@@ -48,9 +48,32 @@ function logDeprecationWarning (deprecatedArgs) {
 }
 
 function logNonDefaultArgsWarning (args) {
+  // cleanly print out the default arguments, allowing for prefix and timestamp
+  function getValueArray (obj, indent = '  ') {
+    if (!_.isObject(obj)) {
+      return [obj];
+    }
+
+    let strArr = ['{'];
+    for (let [arg, value] of _.toPairs(obj)) {
```

Comment:
Shouldn't one use _.isPlainObject verification to make _.toPairs work as expected?